app/src/components/Modals/add-user-modal.tsx

```diff
@@ -27,36 +27,37 @@ import { TFunction } from "i18next";
 interface AddUserModalProps {
   isOpen: boolean;
   onClose: () => void;
-  cityId: string | undefined;
   t: TFunction;
+  userInfo: UserAttributes;
```

Comment:
I think this needs to be marked with a ? (or `| null`) as well, since it'll be `null` while the data is loading from the API.